server/container/core/src/main/java/org/apache/james/core/MimeMessageUtil.java

```diff
@@ -159,24 +159,6 @@ public static void writeMessageBodyTo(MimeMessage message, OutputStream bodyOs)
             } catch (javax.mail.MessagingException ignored) {
                 throw udte;
             }
-        } catch (javax.mail.MessagingException me) {
-            /*
-             * This could be another kind of MessagingException thrown by
-             * MimeMessage.getInputStream(), such as a
-             * javax.mail.internet.ParseException.
-             * 
-             * The ParseException is precisely one of the reasons why the
-             * getRawInputStream() method exists, so that we can continue to
-             * stream the content, even if we cannot handle it. Again, if we get
-             * an exception, we throw the one that caused us to call
-             * getRawInputStream().
-             */
-            try {
```

Comment:
No, let's not disturb too much this code that just demands to break.

This is IDE automatic refactoring, thus safe. I'm not willing to take any risk here.